### ISSUE TITLE
"The directory name is invalid" on Windows

### ISSUE THREAD

**Issue by riverar** [GoogleContainerTools/container-diff#284]
### Expected behavior
container-diff creates valid directories on disk for cache/file storage.

### Actual behavior
container-diff attempts to create folders with names containing colons. This is invalid on Windows.

### Information
- container-diff version: `v0.14.0`
- Operating system: `Windows 10 1809`

### Steps to reproduce the behavior

```
container-diff diff remote://mcr.microsoft.com/windows/servercore:1709_KB4074588 remote://mcr.microsoft.com/windows/servercore:1803 -n
2019/01/08 18:23:26 Unable to read "C:\\Users\\Rafael\\.docker\\config.json": open C:\Users\Rafael\.docker\config.json: The system cannot find the path specified.
2019/01/08 18:23:26 Unable to read "C:\\Users\\Rafael\\.docker\\config.json": open C:\Users\Rafael\.docker\config.json: The system cannot find the path specified.
ERRO[0000] error retrieving image remote://mcr.microsoft.com/windows/servercore:1803: mkdir C:\Users\Rafael\AppData\Local\Temp\mcr.microsoft.comwindowsservercore@sha256:260689daae3d9fb5bfca2881cab21e7ca07ae83849bb0511220b83ec6fe99bd5189428579: The directory name is invalid.
error retrieving image remote://mcr.microsoft.com/windows/servercore:1709_KB4074588: mkdir C:\Users\Rafael\AppData\Local\Temp\mcr.microsoft.comwindowsservercore@sha256:095aa5068b82490433e4a943c4221360294a81223ff487f012434a92d2322f90098830182: The directory name is invalid.
```

**Comment by sandywang1982** [comment]
when will the code with this fix be released?

**Comment by nkubala** [comment]
@sandywang1982 I just released [v0.15.0](https://github.com/GoogleContainerTools/container-diff/releases/tag/v0.15.0)! Please test and make sure everything is fixed :)`
